charts/velero/values.yaml

```diff
@@ -284,8 +284,12 @@ configuration:
   # here if using a non-default value. The `velero server` default values are shown in the
   # comments below.
   # --------------------
+  # `velero server` not empty. kopia or restic
```

Comment:
> `velero server` not empty. kopia or restic

This is an optional field, correct? This sentence looks like we must set it as either kopia or resitc.

Reply:
I've changed it to an optional field, if not set, the default value would be `restic`